src/components/ReleaseMonitor/ReleaseMonitor.tsx

```diff
@@ -0,0 +1,109 @@
+  const [tenantSelections, setTenantSelections] = React.useState<string[]>([]);
+  const [statusSelection, setStatusSelection] = React.useState('');
+  const { namespaces, namespacesLoaded: loaded } = useNamespaceInfo();
+  const releases = useGetAllReleases(loaded, namespaces);
+  const applications = Array.from(new Set(releases.map((release) => release.application)));
+  const tenants = Array.from(new Set(releases.map((release) => release.tenant)));
+
+  const onSearchChange = (value: string) => {
+    setSearchValue(value);
+  };
+
+  const onFilter = React.useCallback(
+    (release: Release) => {
+      let searchValueInput: RegExp;
+      try {
+        searchValueInput = new RegExp(searchValue, 'i');
+      } catch (err) {
+        searchValueInput = new RegExp(searchValue.replace(/[.*+?^${}()|[\]\\]/g, '\\$&'), 'i');
+      }
+
+      const matchesSearchValue = release.releasePlan.search(searchValueInput) >= 0;
+      const matchesTenantValue = tenantSelections.includes(release.tenant);
+      const matchesApplicationValue = applicationSelections.includes(release.application);
+      const matchesStatusValue = release.status.toLowerCase() === statusSelection.toLowerCase();
+
+      return (
+        (searchValue === '' || matchesSearchValue) &&
+        (tenantSelections.length === 0 || matchesTenantValue) &&
+        (applicationSelections.length === 0 || matchesApplicationValue) &&
+        (statusSelection === '' || matchesStatusValue)
+      );
+    },
+    [searchValue, tenantSelections, applicationSelections, statusSelection],
+  );
+
+  const filteredReleases = React.useMemo(() => {
+    return releases.filter(onFilter);
+  }, [releases, onFilter]);
+
+  const onApplicationSelectionsChange = (selections: string[]) => {
+    setApplicationSelections(selections);
+  };
+
+  const onTenantSelect = (itemId: string | number | undefined) => {
+    if (itemId === undefined) {
+      setTenantSelections([]);
+      return;
+    }
+    const itemStr = itemId.toString();
+    setTenantSelections(
+      tenantSelections.includes(itemStr)
+        ? tenantSelections.filter((selection) => selection !== itemStr)
+        : [itemStr, ...tenantSelections],
+    );
+  };
+
+  const onStatusSelect = (itemId: string | number | undefined) => {
+    if (itemId === undefined) {
+      setStatusSelection('');
+      return;
+    }
+    const itemStr = itemId.toString();
+    setStatusSelection(statusSelection === itemStr ? '' : itemStr);
+  };
+
+  const onClearFilters = () => {
+    setSearchValue('');
+    setApplicationSelections([]);
+    setTenantSelections([]);
+    setStatusSelection('');
+  };
+
+  return (
+    <PageLayout title="Release Monitor" description="The dashboard to monitor your cared releases">
+      <PageSection padding={{ default: 'noPadding' }} variant={PageSectionVariants.light} isFilled>
+        <div style={{ backgroundColor: 'white' }}>
```

Comment:
is this `div` really necessary? (specifically this  `backgroundColor: 'white'` style) 🤔

Reply:
That's interesting, remove this works, it used to show some gray layout with some boarder when it's empty record, maybe my new empty component works somehow. Thanks, remove it.